CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## [UNRELEASED] neptune 1.7.1
 
+### Changes
```

Comment:
You have to rebase as the latest version is `1.8.0`

Reply:
fixed